models/marts/fact_execution.yml

```diff
@@ -151,60 +151,6 @@ models:
           hidden: true
         constraints:
           - type: not_null
-      - name: run_started_at
-        description: ''
-        data_type: timestamp
-        meta:
-          label: Run Started At
-          hidden: false
-          datatype: date
-          format:
-            type: date
-            pattern: YYYY-MM-DD HH:mm:ss
-      - name: was_full_refresh
-        description: ''
-        data_type: boolean
-        meta:
-          label: Was Full Refresh
-          datatype: boolean
-      - name: thread_id
-        description: ''
-        data_type: integer
-        meta:
-          label: Thread ID
-          datatype: integer
-      - name: compile_started_at
-        description: ''
-        meta:
-          label: Compile Started At
-          hidden: false
-          datatype: date
-          format:
-            type: date
-            pattern: YYYY-MM-DD HH:mm:ss
-        meta:
-          label: Compile Started At
-          hidden: false
-          datatype: date
-          format:
-            type: date
-            pattern: YYYY-MM-DD HH:mm:ss
-      - name: query_completed_at
-        description: ''
-        meta:
-          label: Query Completed At
-          hidden: false
-          datatype: date
-          format:
-            type: date
-            pattern: YYYY-MM-DD HH:mm:ss
-        meta:
-          label: Query Completed At
-          hidden: false
-          datatype: date
-          format:
-            type: date
-            pattern: YYYY-MM-DD HH:mm:ss
```

Comment:
these should also be removed from [`fact_execution.sql`](https://github.com/flexanalytics/dbt_observability_marts/blob/mart-fixes/models/marts/fact_execution.sql#L58)